### DISCUSSION TITLE
Use AWS

### DISCUSSION THREAD

**Discussion by Giovanni-Petrella** [MohmmedAshraf/laravel-translations#15] [Q&A]
Can use on Amazon AWS with Vapor?

**Comment by MohmmedAshraf** [comment]
Hi @Giovanni-Petrella, 

I think it should be working without any issues give it a try and let me know if you have any issue

**Reply by Giovanni-Petrella** [reply]
Are you sure? we use Vapor and AWS S3 and therefore the filesystem is read-only

**Reply by MohmmedAshraf** [reply]
tbh I have not tried with AWS and I have no idea if this will work or not I will try to grab the free vapor account and give it a shot soon

**Comment by msucevan** [comment]
Update on this issue? Didn't tried yet in Vapor, but is a requirement for my application.

**Comment by Giovanni-Petrella** [comment]
Thank you, for now the translations are only done by the developers. As
soon as it is necessary to have other people do it we will check the
situation




*Giovanni Petrella*
windCommerce - www.windcommerce.com www.bindcommerce.com
Servizi per il Commercio Elettronico
Mob: +39 328 0952051

Centralino: +39 011 08 91 220 - Mob & WhatsApp: +39 328 0952051
<393280952051> -  Skype: prismadigitale
windCommerce è un marchio di bindCommerce s.r.l.


Il giorno ven 16 feb 2024 alle ore 13:14 Marius Sucevan <
***@***.***> ha scritto:
